utilities/tools/Test-ModuleLocally.ps1

```diff
@@ -10,7 +9,7 @@ subscription Id, principal Id, tenant ID and other parameters that need to be to
 .PARAMETER TemplateFilePath
 Mandatory. Path to the Bicep/ARM module that is being tested
 
-.PARAMETER ParameterFilePath
+.PARAMETER TestFilePath
```

Comment:
I'd suggest to continue calling this parameter file path as that is what it is. We won't ever provide a template test file path through this parameter. In the rest of the CI we (should) only have used testfileapath if the underlying file could be a parameter file or a template file

Reply:
I thought aligning terminologies align how our CI and Local tests work. I suspect we'd want to evolve this script when we move to the .bicep file testing mechanism? Not very opinionated here, as alignment was the only driver.

Reply:
Agree. I think we'll remove the `ParameterFilePath`/`TestFilePath` parameter eventually (& adjust the logic accordingly).
For the time being though, I suggest to keep it as is. The alignment of the terminlology is good and important - but only in those cases where we actually refer to the same concept. 

To re-itertate: 
- In the CI environment we're referring to the `TestFilePath` as a placeholder for both classic Parameter Files (that would be invoked in combination with a template file) & the new Bicep Test Files (that would be invoked on their own). 
- In here, this parameter is used exclusively for the classic parameter files and you cannot actually use it for the new Bicep Test Files. Instead you'd provide only the `TemplateFilePath` which would point to on of the new Bicep Test Files (instead of a module template).

It's also how it is implemented in the `New-TemplateDeployment` & `Test-TemplateDeployment` scripts (i.e., they both use the `ParameterFilePath` parameter as it only inteded for Parameter-JSON files anyways).

I hope that made sense :D

Reply:
@MrMCake Reverted to previous naming.

Reply:
@MrMCake shall I resolve?